tex/generic/pgf/utilities/pgfkeys.code.tex

```diff
@@ -26,6 +26,13 @@
 \def\pgfkeys@empty{}
 \long\def\pgfkeys@firstoftwo#1#2{#1}
 \long\def\pgfkeys@secondoftwo#1#2{#2}
+\long\def\pgfkeys@ifcsname#1{%
+  \ifcsname#1\endcsname
+    \expandafter\pgfkeys@firstoftwo
+  \else
+    \expandafter\pgfkeys@secondoftwo
+  \fi
+}
```

Comment:
The `\long` is unnecessary since `pgfkeys` is not ready to accept new paragraphs in key names.

Reply:
I've had a discussion with Skillmon in the TeX.SX chat about this. It is in fact possible to smuggle `\detokenize{\par}` into the key name and although this is a rather academic corner case, it's better to make it `\long`.

Non-`\long` control sequences are actually becoming very rare. In expl3 `cs_new:Npn` defaults to `\long`, `\newcommand` defaults to `\long` and in LuaMetaTeX all `\def`s are automatically `\long` with no way to opt out.

Reply:
> Non-`\long` control sequences are actually becoming very rare. In expl3 `\cs_new:Npn` defaults to `\long`, `\newcommand` defaults to `\long` and in LuaMetaTeX all `\def`s are automatically `\long` with no way to opt out.

Reasonable. I will mark all the three conversations/comments resolved.